plugins/kubernetes/app/models/kubernetes/template_filler.rb

```diff
@@ -396,7 +396,7 @@ def set_hpa_scale_target_name
     # Adding the Release ID to allow us to track the progress of a new release from the UI.
     def set_spec_template_metadata
       release_doc_metadata.each do |key, value|
-        pod_template.dig_fetch(:metadata, :labels)[key] ||= value.to_s.parameterize.tr('_', '-')
+        pod_template.dig_fetch(:metadata, :labels)[key] ||= value.to_s.parameterize.tr('_', '-').slice(0, 63)
```

Comment:
I’d cut last 63 symbols, not first.

Reply:
I meant to do a truncation, so I kinda wanted the first chars ... also the first will include the persons name, which I think is meaningful here